app/_src/quickstart/kubernetes.md

```diff
@@ -54,41 +54,10 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: kuma-demo
-  namespace: kuma-demo
   labels:
     kuma.io/sidecar-injection: enabled
 ```
 
-Run:
-
-```sh
-kubectl get namespace kuma-demo -oyaml
-```
-
-to see what the full namespace looks like:
-
-```sh
-apiVersion: v1
-kind: Namespace
-metadata:
-  annotations:
-    kubectl.kubernetes.io/last-applied-configuration: |
-      {"apiVersion":"v1","kind":"Namespace","metadata":{"labels":{"kuma.io/sidecar-injection":"enabled"},"name":"kuma-demo"}}
-  labels:
-    kuma.io/sidecar-injection: enabled
-  creationTimestamp: "2021-08-13T09:17:48Z"
-  labels:
-    kubernetes.io/metadata.name: kuma-demo
-  name: kuma-demo
-  resourceVersion: "749"
-  uid: 66b1279e-e49c-427d-af01-3adc91e505c1
-spec:
-  finalizers:
-  - kubernetes
-status:
-  phase: Active
-```
```

Comment:
I found this fragment to be just unnecessary. If we want to keep it, let's remove the example above and clean this resource from unnecessary information